src/main/java/org/apache/commons/text/names/HumanNameParser.java

```diff
@@ -131,20 +135,24 @@ public Name parse(String name) {
 
         NameString nameString = new NameString(name);
         // TODO compile regexes only once when the parser is created
+        String salutations = StringUtils.join(this.salutations, " |") + "";
```

Comment:
Righto, so actually the + "" can be removed. 

And we update the salutationRegex to look like:

`String salutationRegex = "^(?i)(("+salutations+")(\\.|\\s))";`

That way we match "dr", "Dr.", "Mrs.", "mr ", etc.